CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
   * Users only have control of the HTTP header when Cortex is not frontend by an auth proxy validating the tenant IDs
 * [CHANGE] Some files and directories created by Mimir components on local disk now have stricter permissions, and are only readable by owner, but not group or others. #4394
 * [CHANGE] Compactor: compactor will no longer try to compact blocks that are already marked for deletion. Previously compactor would consider blocks marked for deletion within `-compactor.deletion-delay / 2` period as eligible for compaction. #4328
+* [CHANGE] Memberlist: forward only changes, not entire original message. #4419
```

Comment:
Why is this listed as a breaking change?

Reply:
It’s a change, but not a breaking one. I guess it could be moved under BUGFIX, as it fixes problem with messages looping in the gossip network forever.